synapse/python_dependencies.py

```diff
@@ -86,6 +86,7 @@
     "saml2": ["pysaml2>=4.5.0"],
     "url_preview": ["lxml>=3.5.0"],
     "test": ["mock>=2.0", "parameterized"],
+    "sentry": ["sentry-sdk>=0.7.2"],
```

Comment:
I'm a bit concerned that we tell people to install with `synapse[all]`, and it's starting to pull in the entire internet. Perhaps we should consider revising that advice.

Reply:
I think I somewhat agree, though I think that is a change to consider outside of this PR

Reply:
yup